public_html/wp-content/plugins/pattern-creator/pattern-creator.php

```diff
@@ -82,9 +98,31 @@ function enqueue_assets() {
 		$post_id = $post->ID;
 	}
 
+	// Update the post id if we're editing a pattern.
+	if ( should_load_creator() && is_editing_pattern() ) {
+		$post_id = get_query_var( PATTERN_ID_VAR );
```

Comment:
You should also reset `$post` here too, since it's used in the `allowed_block_types` filter below.
```suggestion
		$post_id = get_query_var( PATTERN_ID_VAR );
		$post    = get_post( $post_id );
```